src/index.html

```diff
@@ -1,13 +1,284 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>B&O</title>
+    <link
+    rel="icon"
+    type="image/x-icon"
+    href="./images/icons/favicon.ico"
+    >
```

Comment:
![image](https://github.com/mate-academy/layout_landing-page/assets/54860641/7a563641-7da7-4a55-ae55-4b6888e6cbcd)
Fix this in all code.

Reply:
Please fix the comment